dvc/command/version.py

```diff
@@ -129,7 +127,12 @@ def get_supported_remotes():
             if not tree_cls.get_missing_deps():
                 supported_remotes.append(tree_cls.scheme)
 
-        return ", ".join(supported_remotes)
+        if supported_remotes.__len__() == 9:
```

Comment:
@sahilbhosale63, our support for remotes could change, it'd be great to find some way to get all the counts of the remotes that we support.